proto/zds.proto

```diff
@@ -23,11 +37,18 @@ message WorkloadInfo {
   string service_account = 3;
 }
 
+message WindowsNamespace {
+  uint32 id = 1;
+  string guid = 2;
+}
+
 // Add a workload to the ztunnel. this will be accompanied by ancillary data contianing
 // the workload's netns file descriptor.
 message AddWorkload {
   string uid = 1;
   WorkloadInfo workload_info = 2;
+  // The namespace for the workload. Windows only.
+  WindowsNamespace windows_namespace = 3;
```

Comment:
Why not make this an `optional` in WorkloadInfo?

Reply:
Honestly I think it was in an earlier draft of this, but I can't remember why the shift. Agreed that an optional is much cleaner

Reply:
Ah, I remember now, it was because I wanted to model it like the auxiliary data (i.e. the FD) from the unix domain socket communication. I don't really have any preference, and went ahead and just stuck it on workload_info